src/engine/database/mod.rs

```diff
@@ -285,6 +286,7 @@ pub enum CrudOperation {
     UpdateRel(String, String),
     DeleteNode(String),
     DeleteRel(String, String),
+    None
```

Comment:
Add new variant since the type of crud operation is not known in the scope in which before/after request event handlers are fired.